Compiler.coffee

```diff
@@ -70,8 +70,12 @@ class Compiler
         props.className = className
 
       React.cloneElement child, props,
-        React.Children.map child.props.children, (child) =>
-          @traverseChild child, parentClass
+        if React.Children.count(child.props.children) is 1
```

Comment:
Yeah, that makes sense. Could you check, maybe it will work without converting it to array for a single child? e.g.
```
@traverseChild React.Children.toArray(child.props.children)[0],
 +            parentClass
```
replace with
```
@traverseChild child.props.children, parentClass
```